core/src/main/kotlin/org/sertain/Robot.kt

```diff
@@ -205,3 +193,15 @@ public abstract class Robot(vararg subsystems: Subsystem) : IterativeRobot(), Ro
         DISABLED, AUTO, TELEOP
     }
 }
+
+public fun add(lifecycle: RobotLifecycle) {
```

Comment:
Add some docs please 😄 Just something like:

```kotlin
/**
 * @see RobotLifecycle.addListener
 */
```

or whatever the correct `@see` format is. Same thing for the three methods below.